recipe/meta.yaml

```
@@ -76,10 +76,11 @@ requirements:
 
 test:
   commands:
-    - test -f $PREFIX/bin/ncu                      # [linux]
-    - if not exist %LIBRARY_PREFIX%\nsight-compute exit 1  # [win]
+    - test -f $PREFIX/bin/ncu                                                  # [linux]
+    - test -f $PREFIX/bin/ncu-ui                                               # [linux64 or aarch64]
```

The [`aarch64` selector]( https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=aarch64#preprocessing-selectors ) is already Linux specific. So we can drop `linux64`

```suggestion
    - test -f $PREFIX/bin/ncu-ui                                               # [aarch64]
```

---

Wouldn't that skip linux x86_64 though? Trying to leave it enabled for both.

---

Ah sorry. Misread. Yeah let's leave it as-is